client/js/verification-portal/community-health-center/form.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+import { reduxForm } from 'redux-form'
+import injectSheet from 'react-jss'
+
+import RegisterField from '../components/register-field'
+import Button from '../../components/button'
+import Spacer from '../../components/spacer'
+
+const registerFields = inputField => {
+	if (!inputField.name) return inputField.map(field => registerFields(field))
```

Comment:
Just for clarification, what is the purpose of this line?

Reply:
If you will see all form fields group it is only five field groups for now, the second field group addresses it is an array because it has multiple parts, so it needs to loop through it too.